.github/workflows/trigger-test-images.yml

```diff
@@ -0,0 +1,26 @@
+name: Trigger test images
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'exporters/**'
+      - 'charts/**'
+      - 'pelorus-operator/**'
+    types: [opened, reopened, synchronize, closed]
+
+jobs:
+  generate-jobs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/upload-artifact@v3
+        with:
+          name: trigger-information
+          path: ./trigger-information.json
```

Comment:
How does this work?

I see trigger-information that uses path `./` from action `dawidd6/action-download-artifact`, but I don't see where the `trigger-information.json` is being produced.  A comment on how this works in the code would be nice.

Reply:
it is created in `Create json file` step, before uploading it, in this same job